core/src/main/java/io/questdb/cairo/TableUtils.java

```diff
@@ -432,33 +481,60 @@ public static LPSZ iFile(Path path, CharSequence columnName, long columnTxn) {
         return path.$();
     }
 
-    public static boolean isValidColumnName(CharSequence seq) {
-        for (int i = 0, l = seq.length(); i < l; i++) {
-            char c = seq.charAt(i);
+    public static boolean isValidTableName(CharSequence tableName, int fsFileNameLimit) {
+        int l = tableName.length();
+        if (l > fsFileNameLimit) {
+            // Most file systems don't support files
```

Comment:
comment cut in the midde?

Reply:
changed in https://github.com/questdb/questdb/pull/2172/commits/02bc931e7f7ceb0bc15adbb2e09865f152366a01